app/src/main/java/com/kenchen/capstonenewsapp/views/news/NewsViewModel.kt

```diff
@@ -0,0 +1,52 @@
+package com.kenchen.capstonenewsapp.views.news
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.kenchen.capstonenewsapp.App.Companion.remoteApi
+import com.kenchen.capstonenewsapp.model.Article
+import com.kenchen.capstonenewsapp.networking.RemoteResult
+import com.kenchen.capstonenewsapp.utils.mapException
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
+
+class NewsViewModel : ViewModel() {
+
+    private val _headLineNewsLiveData = MutableLiveData<RemoteResult<List<Article>>>()
+    val headLineNewsLiveData: LiveData<RemoteResult<List<Article>>> = _headLineNewsLiveData
+
+    private val test = MutableLiveData<List<String>>()
```

Comment:
Looks like this `test` live data could be removed.

Reply:
Yes, I forgot 🤣